aws-flb-firehose.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: aws-flb-firehose
+  version: 1.7.2
+  epoch: 0
+  description: A Fluent Bit output plugin for Amazon Kinesis Data Firehose
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - wolfi-baselayout
+      - busybox
+      - build-base
+      - go
+      - ca-certificates-bundle
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/aws/amazon-kinesis-firehose-for-fluent-bit
+      tag: v${{package.version}}
+      expected-commit: dcbe1a0191abd6242182af55547ccf99ee650ce9
+
+  - runs: |
+    mkdir -p ${{targets.destdir}}/fluent-bit/
```

Comment:
I think we'll need to write these to `${{targets.destdir}}/usr/bin` and add a compat subpackge that has a symlink to `/fluent-bit/`

Reply:
i.e. similar to this https://github.com/wolfi-dev/os/blob/85f421efeb0c7c93e10ecfd85b0d11169af973e1/keda.yaml#L50-L58

Reply:
Thanks! Ack.